bool enable
float32 throttle_cmd

#not to worry about these things
bool clear #clear driver overrides
bool ignore #ignore driver overrides
